steps/synopsys-vcs-sim/configure.yml

```diff
@@ -0,0 +1,40 @@
+#=========================================================================
+# Synopsys VCS -- Generic VCS Simulation
+#=========================================================================
+
+name: synopsys-vcs-sim
+
+#-------------------------------------------------------------------------
+# Inputs and Outputs
+#-------------------------------------------------------------------------
+
+inputs:
+  - adk
+  - design.vcs.v
+  - testbench.sv
+  - design.sdf
+  - design.args
+
+outputs:
+  - design.vpd
+
+#-------------------------------------------------------------------------
+# Commands
+#-------------------------------------------------------------------------
+
+commands:
+  - |
+    vcs -R -sverilog -top $testbench_name \
+        inputs/*.v inputs/*.sv \
+        inputs/adk/stdcells.v \
+        -timescale=1ns/1ps \
+        +vcs+dumpvars+outputs/design.vpd \
+        $(cat inputs/design.args 2>/dev/null || true)
```

Comment:
I think this is fine, but I remember vcs had a command to take more dash-args from a file. Maybe just -f?